locales/en-US/notes.properties

```diff
@@ -19,7 +19,7 @@ syncError=Sync Error
 
 savingChanges=Saving changes…
 changesSaved=All changes saved
-
+maximumPadSizeExceeded=The multiple notes feature is coming soon. Meanwhile your note reached its maximum size, please cleanup before adding more content to it.
```

Comment:
I think the verb in English is "clean up", while "cleanup" is the noun? Also not sure if there should be a comma before "your note".